app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :users
+  belongs_to :posts
+
+  def update_comments_counter
+    post.increment!(:commentsCounter)
+  end
+end
```

Comment:
- Just like you did in the Post model, please use the `after_save` method to call this function, this would ensure that the commentsCounter is only updated when the comment is saved in the database 😄